src/modules/SX127x/SX1278.h

```diff
@@ -10,6 +10,8 @@
 
 // SX1278 specific register map
 #define RADIOLIB_SX1278_REG_MODEM_CONFIG_3                     0x26
+#define RADIOLIB_SX1278_REG_DIO_MAPPING_1                      0x40
+#define RADIOLIB_SX1278_REG_DIO_MAPPING_2                      0x41
```

Comment:
This (and actually all of the changes in SX1278.h/.cpp) should be done in SX127x files, to be common to all the SX127x modules (unless there's a specific incompatibility between SX1278/76 and 72/73.

Reply:
Thing is, I started with the chip I have here and went down the inheritance chain, but not up.
That being said, I checked and you are right, SX1278 and SX1273 have the same behavior when it comes to DIO pins, so I moved the required support code to the SX127x.h/.cpp files in the latest commit.